### PR TITLE
🦺[Tech debt] Add card disclaimer toast 

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Overview 
- Adds a alert/toast that shows when the user taps on `Card`
- Adds localizations

## Screenshots
|Clip|
|-----|
|carddisclamer|

### CODE REVIEW

File: loafwallet/src/Strings/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -367,6 +367,18 @@
 /* Card balance */
 "LitecoinCard.cardBalance" = "Solde de la carte";
 
+/* Features and limitations */
+"LitecoinCard.Disclaimer.bullets" = "- Nouvelle inscription\n- Connexion (compte existant)\n- Aucun solde affiché\n- Pas de transfert\n- Seulement nous";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/fr.lproj/Localizable.strings:371]: Wrong translation. Replace "Seulement nous" -> "États-Unis uniquement"

File: loafwallet/src/Strings/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -368,6 +368,18 @@
 /* Card balance */
 "LitecoinCard.cardBalance" = "Balance de tarjeta";
 
+/* Features and limitations */
+"LitecoinCard.Disclaimer.bullets" = "- Nuevo registro\n- Ingrese cuenta existente)\n- No se muestra saldo\n- Sin transferencia\n- Solo nosotros";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/es.lproj/Localizable.strings:372]: Wrong translation: "Estados Unidos solamente" is more suited than "Solo nosotros"

File: loafwallet/src/Strings/pt.lproj/Localizable.strings [hunk 3]
```diff
@@ -368,6 +368,18 @@
 /* Card balance */
 "LitecoinCard.cardBalance" = "Saldo do cartão";
 
+/* Features and limitations */
+"LitecoinCard.Disclaimer.bullets" = "- Novo registro\n- Entrar em uma conta existente)\n- Nenhum saldo mostrado\n- Sem transferência\n- Apenas nós";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/pt.lproj/Localizable.strings:372]: Wrong translation: "Apenas estados unidos" is more suited than "Apenas nós"

**Review by mosadialiou — CHANGES_REQUESTED** [review]

**Comment by mosadialiou** [comment]
@kcw-grunt Make sure all the changes in this commit https://github.com/litecoin-foundation/loafwallet-android/commit/9c0a5305278385df5df1ad9aa5cbff6148149469 are done!

File: loafwallet/src/Strings/fr.lproj/Localizable.strings (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/src/Strings/fr.lproj/Localizable.strings:371]: D'accord.  Merci beaucoup

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**